prow/scripts/skr-svcat-migration.bash

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Script to run SKR svcat migration
+
+set -euo pipefail
+
+readonly SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+readonly TEST_INFRA_SOURCES_DIR="$(cd "${SCRIPT_DIR}/../../" && pwd)"
+
+# shellcheck source=prow/scripts/lib/log.sh
+source "${TEST_INFRA_SOURCES_DIR}/prow/scripts/lib/log.sh"
+# shellcheck source=prow/scripts/lib/utils.sh
+source "${TEST_INFRA_SOURCES_DIR}/prow/scripts/lib/utils.sh"
+
+if [[ "${BUILD_TYPE}" == "pr" ]]; then
```

Comment:
jobguard doesn't work for periodic jobs
I think this script can be shortened to one line:
`make -C tests/fast-integration ci-skr-svcat-migration`
This way we could put it directly in the entrypoint of the prowjob:
```
command: make
args:
- -C
- tests/fast-integration
- ci-skr-svcat-migration
```

Reply:
perfect, thanks. I was told to use skr-integration.sh as a template but your proposal looks much neater.
https://github.com/kyma-project/test-infra/blob/f4478be4f8beae7b1a7e714a99425285555550f0/prow/scripts/skr-integration.sh#L1-L24